### DISCUSSION TITLE
如何在Stencil的节点中添加删除按钮，点击后删除对应节点？

### DISCUSSION THREAD

**Discussion by chenlong-io** [antvis/X6#3985] [Q&A]
需求：用户可以在Stencil中点击删除按钮删除节点。

![111 pic](https://github.com/antvis/X6/assets/10387532/a8a9d9dd-3c3b-408c-8344-3781e3137ead)

上图是在 [官网示例](https://x6.antv.antgroup.com/examples/showcase/practices/#flowchart) 中添加了删除按钮(红框中的代码)

但鼠标移上去没有任何反应，按钮显示不出来，只有鼠标按下后才显示，但此时的节点处于可拖拽状态。
如何让鼠标移到 Stencil 中的节点上时，显示删除按钮，并点击可以删除节点。

**Comment by z77z** [comment]
这个功能很有必要
